### ISSUE TITLE
TypeError: partials is undefined

### ISSUE THREAD

**Issue by mjacinto7777** [redux-utilities/redux-actions#315]
After upgrading to version 2.6.1, my application started to display a blank page with the following error in the console:

**TypeError: partials is undefined**

This error occurs in the lodash/_createWrap.js file: 28.

What could be causing this error?

**Comment by timche** [comment]
Can you post the full error stack please?

**Comment by mjacinto7777** [comment]
TypeError: partials is undefined[Learn More] _createWrap.js:28
wrapper
_createWrap.js:28
suggest
customPropTypes.js:92
./node_modules/semantic-ui-react/dist/commonjs/elements/Icon/Icon.js
Icon.js:147
__webpack_require__
bootstrap:787
fn
bootstrap:150
./node_modules/semantic-ui-react/dist/commonjs/elements/Icon/index.js
index.js:15
__webpack_require__
bootstrap:787
fn
bootstrap:150
./node_modules/semantic-ui-react/dist/commonjs/collections/Message/Message.js
Message.js:40
__webpack_require__
bootstrap:787
fn
bootstrap:150
./portal/utils/Loadable.js/<
https://client-app:4443/build/portal/index.js:2936:117
./portal/utils/Loadable.js
https://client-app:4443/build/portal/index.js:2935:29
__webpack_require__
bootstrap:787
fn
bootstrap:150
./portal/components/modules.js/<
https://client-app:4443/build/portal/index.js:2033:118
./portal/components/modules.js
https://client-app:4443/build/portal/index.js:2033:29
__webpack_require__
bootstrap:787
fn
bootstrap:150
./portal/redux-state/reduxMain.js/<
https://client-app:4443/build/portal/index.js:2624:77
./portal/redux-state/reduxMain.js
https://client-app:4443/build/portal/index.js:2599:29
__webpack_require__
bootstrap:787
fn
bootstrap:150
./portal/redux-state/createStore.js/<
https://client-app:4443/build/portal/index.js:2530:68
./portal/redux-state/createStore.js
https://client-app:4443/build/portal/index.js:2522:29
__webpack_require__
bootstrap:787
fn
bootstrap:150
./portal/index.js/<
https://client-app:4443/build/portal/index.js:2090:81
./portal/index.js
https://client-app:4443/build/portal/index.js:2084:29
__webpack_require__
bootstrap:787
fn
bootstrap:150
0
https://client-app:4443/build/portal/index.js:3386:1
__webpack_require__
bootstrap:787
checkDeferredModules
bootstrap:45
<anonymous>
bootstrap:925
<anonymous>
https://client-app:4443/build/portal/index.js:1:11

**Comment by timche** [comment]
It doesn't mention redux-actions. Can you please re-install node_modules and also re-check that it's caused by the upgrade? From which version did you upgrade?

**Comment by timche** [comment]
Can you try to add a `console.log('==>', partials)` below [there](https://github.com/redux-utilities/redux-actions/blob/master/src/utils/flattenWhenNode.js#L19)?

**Comment by mjacinto7777** [comment]
I was able to find the error.
Until version 2.4.0, the lodash and lodash-es packages were used and in the configurations of the plugin lodash-webpack-plugin I used the following configurations:

new LodashModuleReplacementPlugin ({
	shorthands: true,
	collections: true
})

However, due the way lodash was imported in redux-actions, it had never been a problem, in particular, to semantic-ui-react.
I then checked, that I would have to enable the feature of "curry" methods, which solved my problem.

**new LodashModuleReplacementPlugin ({
	shorthands: true,
	collections: true,
	currying: true
})**

I apologize for supposing that the problem was redux-actions, but the way redux-actions used lodash, it masked a configuration error in lodash-webpack-plugin.

Thank you
